ci-operator/step-registry/openshift/e2e/cert-rotation/conf/mirror/openshift-e2e-cert-rotation-conf-mirror-commands.sh

```diff
@@ -178,26 +189,16 @@ EOZ
 kubectl apply -f /tmp/patch.yml
 kubectl -n kube-system delete pod -l kubernetes.io/minikube-addons=registry
 
-# Enable registry access from VMs
-firewall-cmd --add-port=5000/tcp --zone=internal --permanent
-firewall-cmd --add-port=5000/tcp --zone=public   --permanent
-firewall-cmd --add-port=5000/tcp --zone=libvirt   --permanent
-# FIXME: open access to assisted-service too
-firewall-cmd --add-port=6000/tcp --zone=internal --permanent
-firewall-cmd --add-port=6000/tcp --zone=public   --permanent
-firewall-cmd --add-port=6000/tcp --zone=libvirt   --permanent
-firewall-cmd --reload
-
-
 # Wait for registry to come up again
 retries=0
-export LOCAL_REG="${HOSTNAME}:5000"
+export LOCAL_REG="${REGISTRY_HOSTNAME}:80"
 set +e
 while ! curl -u test:test https://"${LOCAL_REG}"/v2/_catalog && [ $retries -lt 10 ]; do
   if [ $retries -eq 9 ]; then
     exit 1
   fi
   (( retries++ ))
+  sleep 10
```

Comment:
I'm curious what is the reason for this sleep? Avoid DDOSing the registry?

Reply:
Yes, we make 10 rapid burst requests, so it does a pause in between to let the registry initialize meanwhile